tiatoolbox/dataloader/wsireader.py

```diff
@@ -0,0 +1,101 @@
+"""
+WSIReader for WSI reading or extracting metadata information from WSIs
+"""
+import pathlib
+import numpy as np
+import yaml
+from PIL import Image
+
+import openslide
+
+
+class WSIReader:
+    def __init__(
+        self,
+        input_dir=pathlib.Path.cwd(),
+        file_name=None,
+        output_dir=pathlib.Path(pathlib.Path.cwd(), "output"),
+        tile_objective_value=20,
+        tile_read_size_w=5000,
+        tile_read_size_h=5000,
+    ):
+        """
+        WSI Reader class to read WSI images
+        Args:
```

Comment:
Should we have space between docstring description and then args? The spacing is not consistent.

Reply:
There will be a space as in Google style docstring. Corrected this in https://github.com/TIA-Lab/tiatoolbox/commit/b49ec891c499a589134dd43e5786bd0a7d8d39ff